### PR TITLE
Update type of connector.sync frequency to string

### PR DESCRIPTION
**Pull request by fivetran-jovanmanojlovic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.9.0...HEAD)
+## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.9.1...HEAD)
+
+## [0.9.0](https://github.com/fivetran/go-fivetran/compare/v0.8.9...v0.9.0)
+
+## Fixed
+- Update type of connector.SyncFrequency to string
```
Comment (beevital) [inline CHANGELOG.md:13]:
```suggestion
- Allow to pass `nil` into ConnectorCreateService.SyncFrequency and ConnectorModifyService.SyncFrequency.
```

**Comment by fivetran-aleksandrboldyrev** [comment]
Please rebase, cause current actual version is 0.9.0

**Review by beevital — COMMENTED** [review]
Add e2e test with `nil` value.
Fix example.

**Review by fivetran-aleksandrboldyrev — APPROVED** [review]
LGTM
